packages/api/src/hooks/useAccountsList.ts

```diff
@@ -39,12 +39,22 @@ const useAccountsList = () => {
     // Add balance to each account
     const modified_accounts_with_balance = useMemo(
         () =>
-            modified_accounts?.map(account => ({
-                ...account,
-                /** The balance of the account. */
-                balance: balance_data?.accounts?.[account.loginid]?.balance || 0,
-            })),
-        [balance_data?.accounts, modified_accounts]
+            modified_accounts?.map(account => {
+                const balance = balance_data?.accounts?.[account.loginid]?.balance || 0;
+
+                return {
+                    ...account,
+                    /** The balance of the account. */
+                    balance,
+                    /** The balance of the account in currency format. */
+                    display_balance: Intl.NumberFormat(authorize_data?.preferred_language || 'en-US', {
+                        minimumFractionDigits: account.currency_config?.fractional_digits || 2,
+                        maximumFractionDigits: account.currency_config?.fractional_digits || 2,
+                        minimumIntegerDigits: 2,
```

Comment:
```suggestion
minimumIntegerDigits: 1,
```
could cause an issue with 1.04 formatted to 01.04